scripts/build

```diff
@@ -14,12 +14,12 @@ fi
 
 echo "The repo name is: " $2
 
+./IoTHubManager/run.sh "IoT Hub Manager tests"
```

Comment:
why was this moved?

Reply:
Nothing much. I wanted to test my build failure.